src/main/scala/net/psforever/actors/session/SessionActor.scala

```diff
@@ -6823,7 +6823,7 @@ class SessionActor(middlewareActor: typed.ActorRef[MiddlewareActor.Command], con
         //silo capacity
         sendResponse(PlanetsideAttributeMessage(amenityId, 45, silo.CapacitorDisplay))
         //warning lights
-        sendResponse(PlanetsideAttributeMessage(silo.Owner.GUID, 47, if (silo.LowNtuWarningOn) 1 else 0))
+        sendResponse(PlanetsideAttributeMessage(silo.Owner.GUID, 47, silo.LowNtuWarningOn))
```

Comment:
Is sending a boolean going to work for `PSAM`?

Reply:
Check out the `implicit` `WorldSession.boolToInt` function.